### PR TITLE
Display fully vaccinated status

### LINKED ISSUE #132: [COVAX demo app] Display fully vaccinated status on Profile view
- [x] Display fully vaccinated status when patient receives 2nd dose
- [x] Disable `Record vaccine` button when 2nd dose is administered

### PR DESCRIPTION
**Pull request by Rkareko**
Resolves #132

### CODE REVIEW

File: android/app/src/main/java/org/smartregister/fhircore/fragment/PatientDetailFragment.kt [hunk 1]
```diff
@@ -74,42 +86,91 @@ class PatientDetailFragment : Fragment() {
...
-      {
-        adapter.submitList(it)
-        if (it.isNotEmpty()) {
-          val tvNoVaccinePlaceholder =
-            rootView.findViewById<TextView>(R.id.no_vaccination_placeholder)
-          tvNoVaccinePlaceholder.visibility = View.GONE
-          val noVaccinePlaceholder =
-            rootView.findViewById<View>(R.id.view_vaccination_status_separator)
-          noVaccinePlaceholder.visibility = View.GONE
+    viewModel
```
Comment (rkodev) [inline android/app/src/main/java/org/smartregister/fhircore/fragment/PatientDetailFragment.kt:153]: This code block is not necessary, lines 92-111 will automatically update the UI once search is executed.

**Review by rkodev — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/smartregister/fhircore/viewmodel/PatientListViewModel.kt [hunk 2]
```diff
@@ -72,6 +72,10 @@ class PatientListViewModel(application: Application, private val fhirEngine: Fhi
     MutableLiveData<List<Immunization>>()
   }
 
+  fun getImmunizations(): MutableLiveData<List<Immunization>> {
```
[the remark this answers is not in the document]
Comment (Rkareko) [reply android/app/src/main/java/org/smartregister/fhircore/viewmodel/PatientListViewModel.kt]: updated

File: android/app/src/main/java/org/smartregister/fhircore/activity/PatientDetailActivity.kt [hunk 3]
```diff
@@ -62,6 +62,7 @@ class PatientDetailActivity : AppCompatActivity() {
           putExtra(QuestionnaireActivity.QUESTIONNAIRE_TITLE_KEY, "Record Vaccine")
           putExtra(QuestionnaireActivity.QUESTIONNAIRE_FILE_PATH_KEY, "record-vaccine.json")
           putExtra(PATIENT_ID, fragment.patitentId)
+          putExtra(IS_FIRST_DOSE_ADMINISTERED, fragment.isFirstDoseAdministered)
```
[the remark this answers is not in the document]
Comment (Rkareko) [reply android/app/src/main/java/org/smartregister/fhircore/activity/PatientDetailActivity.kt]: updated

**Review by rkodev — APPROVED** [review]

File: android/app/src/main/java/org/smartregister/fhircore/fragment/PatientDetailFragment.kt (hunk 1 above)
Comment (Rkareko) [reply android/app/src/main/java/org/smartregister/fhircore/fragment/PatientDetailFragment.kt]: updated

**Pull request merged**